src/main/java/com/yahoo/sketches/theta/DirectIntersection.java

```diff
@@ -20,15 +20,15 @@
  * @author Kevin Lang
  */
 class DirectIntersection extends SetOperation implements Intersection {
-  private final short seedHash_;
```

Comment:
I realize that the Google style guides and the Java religious don't like underscores.  Well, that is their opinion based on someone's concept of purity.  Tough!   I use underscores to clearly identify class members vs. local variables.  This avoids accidental name hiding and to avoid having to use _this._, which adds a lot of noise to the code.  All these underscore changes are rejected.